src/main/java/nextstep/qna/domain/Answers.java

```diff
@@ -0,0 +1,41 @@
+package nextstep.qna.domain;
+
+import nextstep.qna.CannotDeleteException;
+import nextstep.users.domain.NsUser;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Answers {
+
+    private List<Answer> answers;
+
+    public Answers() {
+        answers = new ArrayList<>();
```

Comment:
```suggestion
        this(new ArrayList<>());
```

부 생성자는 항상 주 생성자를 호출하는 방식으로 구현한다.